.github/pull_request_template.md

```diff
@@ -1,62 +1,87 @@
-**What this PR does / why we need it:**
+<!-- Choose you PR title carefully as it will be used as the entry in the changelog! -->
 
-<!-- use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged-->
-**Which issue this PR fixes:** fixes #
+### What kind of PR is this?
 
-**Public facing documentation PR** *(if applicable)*
-<!-- https://github.com/elastisys/compliantkubernetes/pull/ -->
+**Required**: Mark one of the following that is applicable:
+- [ ] kind/feature       <!-- This PR adds a new feature -->
```

Comment:
```suggestion
**Required**: Mark one of the following that is applicable:
- [ ] BREAKING CHANGE
- [ ] kind/feature       <!-- This PR adds a new feature -->
```

I am thinking to better highlight changes that could break user applications

Reply:
That would be what `kind/dev-change` is for, and similarly `kind/admin-change` for platform admins. With one of those checked the additional sections "Application Developer notice" respectively "Platform Administrator notice" becomes mandatory.

Reply:
:thinking: That was not super clear to me either

Maybe this
> Optional: Mark one or more of the following that are applicable:

could be changed to clarify that

Reply:
Would this improve the situation?

---

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes must be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes must be marked with `kind/security`

- [ ] kind/admin-change  <!-- This PR introduces an admin facing change, add "Platform Administrator notice" section -->
- [ ] kind/dev-change    <!-- This PR introduces a dev facing change, add "Application Developer notice" section -->
- [ ] kind/security      <!-- This PR introduces a critical security fix, add "Security notice" section -->
- [ ] [kind/adr]()       <!-- This PR implements an ADR, add the link -->

Reply:
I like that :+1:

Reply:
perfekt!